frontend/public/components/utils/datetime.ts

```diff
@@ -9,6 +9,7 @@ export const maxClockSkewMS = -60000;
 export const timeFormatter = new Intl.DateTimeFormat(getLastLanguage() || undefined, {
   hour: 'numeric',
   minute: 'numeric',
+  hour12: true,
```

Comment:
We want to use continue using 24 hour time in locales where that is preferred (which is most of the world :)